runs/comparisons/suprb_tuning.py

```diff
@@ -81,49 +83,91 @@ def suprb_ES_GA_space(trial: Trial, params: Bunch):
             'rule_generation__mutation__sigma', *sigma_space)
         params.rule_generation__init__fitness__alpha = trial.suggest_float(
             'rule_generation__init__fitness__alpha', 0.01, 0.2)
-        #params.rule_generation__operator = trial.suggest_categorical(
+        # params.rule_generation__operator = trial.suggest_categorical(
         #    'rule_generation__operator', ['&', ',', '+'])
 
-        #if params.rule_generation__operator == '&':
+        # if params.rule_generation__operator == '&':
         #    params.rule_generation__delay = trial.suggest_int('rule_generation__delay', 10, 100)
 
-        #params.rule_generation__mutation = trial.suggest_categorical(
+        # params.rule_generation__mutation = trial.suggest_categorical(
         #    'mutation', ['Normal', 'HalfnormIncrease', 'UniformIncrease'])
-        #params.rule_generation__mutation = getattr(mutation, params.rule_generation__mutation)()
+        # params.rule_generation__mutation = getattr(mutation, params.rule_generation__mutation)()
 
         # GA
         params.solution_composition__selection = trial.suggest_categorical(
             'solution_composition__selection',
             ['RouletteWheel',
              'Tournament',
              'LinearRank', 'Random'])
-        params.solution_composition__selection = getattr(ga.selection, params.solution_composition__selection)()
+        params.solution_composition__selection = getattr(
+            ga.selection, params.solution_composition__selection)()
 
         if isinstance(params.solution_composition__selection, ga.selection.Tournament):
-            params.solution_composition__selection__k = trial.suggest_int('solution_composition__selection__k', 3, 10)
+            params.solution_composition__selection__k = trial.suggest_int(
+                'solution_composition__selection__k', 3, 10)
 
         params.solution_composition__crossover = trial.suggest_categorical('solution_composition__crossover',
                                                                            ['NPoint', 'Uniform'])
-        params.solution_composition__crossover = getattr(ga.crossover, params.solution_composition__crossover)()
+        params.solution_composition__crossover = getattr(
+            ga.crossover, params.solution_composition__crossover)()
 
         if isinstance(params.solution_composition__crossover, ga.crossover.NPoint):
-            params.solution_composition__crossover__n = trial.suggest_int('solution_composition__crossover__n', 1, 10)
+            params.solution_composition__crossover__n = trial.suggest_int(
+                'solution_composition__crossover__n', 1, 10)
 
         params.solution_composition__mutation__mutation_rate = trial.suggest_float(
             'solution_composition__mutation_rate', 0, 0.1)
```

Comment:
is this a good range given past tuning results?

Reply:
The range is overall fine, although the most tuning results are usually between 0 and 0.06. I think it is fine to keep the range as it is